tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -78,6 +80,36 @@ def _run(self):
             )
         )
 
+        # Check if monitor key exists, if not sync
+        try:
+            if NS._int.client.read(
+                "clusters/%s/_mon_key" % NS.tendrl_context.integration_id
+            ):
+                pass
```

Comment:
We need to continue with other syncing stuff. If `_mon_key` just dont try to write it again.